install_handler.js

```diff
@@ -21,7 +22,7 @@ function listInstalledVersions(){
 
         const versions = getVersionData().versions;
 
-        const directories = getDirectories(installPath);
+        const directories = getDirectories(insDirs.installedDir);
```

Comment:
This function causes an error for me:
```
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be one of type string, Buffer, or URL. Received type object
```
At least I think that comes from here. That's in the space where the installed versions should be listed.